tokens-studio/figma-plugin
# Deduplicate resolve modal fixes

### Why does this PR exist?

#2779 




### What does this pull request do?

- Move resolve duplicate warning button
- Fix duplicate resolve modal copy



### Testing this change

- Switch to production version
- Create a group with a conflicting/overlapping token name, but unique group name
- Duplicate this new group to an existing group name, with an overlapping token
- Switch back to this version
- Check the icon button UI and the modal UI



### Additional Notes (if any)

<img width="498" alt="image" src="https://github.com/user-attachments/assets/86a97c5a-51de-423e-b62e-0db700aa91ec">

<img width="498" alt="image" src="https://github.com/user-attachments/assets/bfc0c3e8-c998-4a07-aadb-8049b7fb91cf">

<img width="496" alt="image" src="https://github.com/user-attachments/assets/b86f090e-2b34-4ef9-bc7f-04661205feaf">
